src/components/panel/panel.js

```diff
@@ -129,7 +129,7 @@ angular
  *     Only disable if focusing some other way, as focus management is required for
  *     panels to be accessible. Defaults to true.
  *
- * @returns {$MdPanelReference} panelRef
+ * @returns {$mdPanelReference} panelRef
```

Comment:
When using a `$`, I think it would be easy to mistake the value for a singleton that you might inject anywhere. I also want to follow the widely-accepted JS standard that constructors / types are PascalCase.

@ThomasBurleson do you have a strong counter to this?